autoPyTorch/api/base_task.py

```diff
@@ -551,89 +551,150 @@ def _do_dummy_prediction(self, num_run: int) -> None:
                     % (str(status), str(additional_info))
                 )
 
-    def _do_traditional_prediction(self, num_run: int, time_for_traditional: int) -> int:
+    def _do_traditional_prediction(self, num_run: int, time_left: int, func_eval_time_limit_secs: int
+                                   ) -> int:
+        """
+        Fits traditional machine learning algorithms to the provided dataset, while
+        complying with time resource allocation.
+
+        This method currently only supports classification.
+
+        Args:
+            num_run: (int)
+                An identifier to indicate the current machine learning algorithm
+                being processed
```

Comment:
Does `num_run` mean like the job id of a given ML algorithm?
I did not get this variable `num_run`.

Reply:
Yes exactly, it is just an ordinal number to track how many machine learning algorithms we have launched, so this is the expectation:
- Num_run 0 is the dummy prediction, in case all run fails, to be at least as good as random prediction.
- Num_run 1-7 (depends on the number of traditional pipelines) should be traditional pipelines
- then all remaining runs are neural networks.

I will add this to #127